clevertap-core/src/main/java/com/clevertap/android/sdk/network/NetworkManager.java

```diff
@@ -826,9 +839,15 @@ private void setFirstRequestTimestampIfNeeded(int ts) {
     @WorkerThread
     private void setSpikyDomain(final Context context, String spikyDomainName) {
         logger.verbose(config.getAccountId(), "Setting spiky domain to " + spikyDomainName);
-        StorageHelper.putString(context, StorageHelper.storageKeyWithSuffix(config, Constants.SPIKY_KEY_DOMAIN_NAME),
-                spikyDomainName);
-        ctApiWrapper.getCtApi().setSpikyDomain(spikyDomainName);
+        StorageHelper.putString(context, StorageHelper.storageKeyWithSuffix(config, Constants.SPIKY_KEY_DOMAIN_NAME), spikyDomainName);
+        ctApiWrapper.getCtApi().setCachedSpikyDomain(spikyDomainName);
+    }
+
+    @WorkerThread
+    private void setCustomHandshakeDomain(final Context context, String customDomain) {
+        logger.verbose(config.getAccountId(), "Setting spiky domain to " + customDomain);
```

Comment:
This log should say custom domain instead of spiky domain

Reply:
done